pkg/fw/Dockerfile

```diff
@@ -172,6 +178,34 @@ COPY --from=build /rtw88/*.bin /lib/firmware/rtw88/
 COPY --from=build /lib/firmware/intel/ice /lib/firmware/intel/ice
 COPY --from=build /lib/firmware/hailo /lib/firmware/hailo
 
+ARG TARGETARCH
+
+# Remove unnecessary firmware based on TARGETARCH and print freed space
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+    initial_size=$(du -sm /lib/firmware | cut -f1); \
+    rm -rf /lib/firmware/nvidia \
+           /lib/firmware/ti-connectivity \
+           /lib/broadcom \
+           /lib/firmware/*t234* \
+           /lib/firmware/*t194* \
+           /lib/firmware/nvpva* \
+           /lib/firmware/rtw88 ; \
```

Comment:
This is Realtek firmware for a PCIe WiFi card, this is not strictly related to arm64...

Reply:
I tried my best to understand if the firmware can appear on a given architecture (I realize it's a common situation when arm chips are a part of a device with intel CPU and vise versa). But it's better to double check . I hoped for your review here ;)

Reply:
Fixed